pkg/vcs/linux.go

```diff
@@ -160,10 +160,12 @@ func (ctx *linux) EnvForCommit(
 
 func linuxClangPath(tags map[string]bool, binDir, defaultCompiler string) string {
 	version := ""
+	// The defaultCompiler and clang-15 are assumed to be available.
 	switch {
-	case tags["v5.9"]:
-		// Verified to work with 14.0.6.
+	case tags["v6.15"]:
 		return defaultCompiler
+	case tags["v5.9"]:
+		return "clang-15"
```

Comment:
IIRC we had some special bisect_bin dir with various versions of the compiler. Do we use only that dir, or default PATH as well? After this change we need clang-15 to be present in either of these locations. Do we have it there? I don't see the docker file install clang-15 now, and we probably don't have clang-15 in existing bisect_bin.